sklearn/ensemble/_hist_gradient_boosting/tests/test_gradient_boosting.py

```diff
@@ -409,6 +412,20 @@ def test_infinite_values():
     np.testing.assert_allclose(gbdt.predict(X), y, atol=1e-4)
 
 
+def test_consistent_lengths():
+    X = np.array([-np.inf, 0, 1, np.inf]).reshape(-1, 1)
```

Comment:
remove nans?

Reply:
what do you mean? don't understand

Reply:
sorry meant np.inf but that's fine, you can ignore